packages/fe/components/searchbar.vue

```diff
@@ -12,12 +12,12 @@
       :class="['searchbar', showTypeahead ? 'has-typeahead' : 'no-typeahead', `theme__${theme}`, { empty, loading }]">
 
       <FieldContainer
-        field-key="typeahead"
+        :field-key="fieldKey"
         :scaffold="{
           type: 'typeahead',
           inputType: 'text',
           modelKey: 'typeahead',
-          placeholder: `Search ${datasetListTypeahead.length || '...'} datasets`,
+          placeholder: inputPlaceholder,
```

Comment:
Let's change the placeholder text for the CID search to:

`Search by filename or CID`

since that's what we're actually searching by for now. In the future we'll tweak this again.